AudioManagerUtilities/src/CAmSocketHandler.cpp

```diff
@@ -423,19 +457,35 @@ am::am_Error_e CAmSocketHandler::addFDPoll(const int fd, const short event, IAmS
   */
 am_Error_e CAmSocketHandler::removeFDPoll(const sh_pollHandle_t handle)
 {
+    CHECK_CALLER_THREAD_ID()
+    
     VectorListPoll_t::iterator iterator = mListPoll.begin();
-
+    
     for (; iterator != mListPoll.end(); ++iterator)
     {
         if (iterator->handle == handle)
         {
             iterator = mListPoll.erase(iterator);
             mSetPollKeys.pollHandles.erase(handle);
             mRecreatePollfds = true;
-            return (E_OK);
+            break;
         }
     }
-    return (E_UNKNOWN);
+    
+    if (iterator == mListPoll.end())
+        return (E_UNKNOWN);
+    
+    VectorListPoll_t::iterator iteratorActivePolls = mListActivePolls.begin();
+    for (; iteratorActivePolls != mListActivePolls.end(); ++iteratorActivePolls)
```

Comment:
Related to the name -  there seems little need to create a new iterator variable in the same scope.  You could have reused the same one.  But for this loop, since it is used locally only, I think even better style is to declare it as locally as possible, which is within the loop scope:
```
for (auto it = mListActivePolls.begin(); it != mListActivePolls.end(); ++it)
```

By the way, assuming C++11 is acceptable in the project then of course range-based for would the nicest way to write this.